azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/features/JobApi.java

```diff
@@ -37,5 +43,14 @@
    @GET
    @ResponseParser(ParseJobStatus.class)
    JobStatus jobStatus(@EndpointParam URI jobURI);
+
+   /**
+    * Get status of captured custom image after capture call
+    */
+   @GET
+   @Fallback(Fallbacks.NullOnNotFoundOr404.class)
```

Comment:
Try to avoid returning null collections; it is a font of bugs and NPEs. Change the fallback to return an empty list.